content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -132,7 +132,9 @@ stages:
       - performance.json
   training:
     desc: train your model
-    cmd: python train.py
+    cmd:
+      - echo "Train my model!"
+      - python train.py
```

Comment:
@skshetry can you think of a better example? I added an echo command here for now but a good example would show the usefulness of having several `cmd`s. TBH I'm not sure what the original motivation for this change was so I can't think of something off the top of my head, hoping you are more familiar. Thanks

Reply:
What about this?
```suggestion
    cmd:
      - pip install -r requirements.txt
      - python train.py
```

Reply:
LGTM!